tests/testthat/test-zzz-e2e.R

```diff
@@ -86,6 +86,8 @@ test_that("can run model, get status and results", {
   expect_gt(length(results_data$interventions), 0)
   expect_gt(length(results_data$time_series), 0)
 
+  expect_gt(length(results_data$time_series$vaccinated), 0)
```

Comment:
I'm wondering whether this expectation could be more specific - we know this time series has to be the same length as the `time_end` or simulation time.

Reply:
Is `time_end` a constant in the package, or is it in the result data (which we don't have direct access to in the e2e test)?

Reply:
I can at least put in an extra check that all the expected time series are the same length.